crates/adapters/src/integrated/delta_table/test.rs

```diff
@@ -711,12 +711,12 @@ proptest! {
         // Filter delta table by id
         let mut json_file_filtered_by_id = delta_table_snapshot_to_json::<TestStruct2>(
             &table_uri,
-            &HashMap::from([("snapshot_filter".to_string(), "id >= 10000 and id <= 20000".to_string())]));
+            &HashMap::from([("snapshot_filter".to_string(), "id >= 10000 ".to_string())]));
```

Comment:
why change this test?

Reply:
This is the regression I mentioned in the second commit.